tests/core/test_array.py

```diff
@@ -2,6 +2,26 @@
 import numpy.testing as npt
 
 
+def broadcast_first():
+    from glass.core.array import broadcast_first
```

Comment:
I've always put test imports at the top of the file - as have most ARC projects I've worked on. Why put it here?

Reply:
Other already existing tests follow this pattern and I was trying to maintain uniformity. Should I update the import style throughout the tests in a separate PR?

Reply:
Makes sense. I think leave it as you have it, and can you raise an issue just to discuss the style?